addons/stock/views/stock_rule_views.xml

```diff
@@ -103,6 +103,13 @@
                                 <field name="partner_address_id" attrs="{'invisible': [('action', '=', 'push')]}"/>
                                 <field name="delay"/>
                             </group>
+                            <group name="scheduling_group" string="Scheduling" attrs="{'invisible': [('action', '=', 'push')]}" groups="base.group_no_one">
+                                <field name="propagate_date"/>
+                                <label for="propagate_date_minimum_delta" attrs="{'invisible': [('propagate_date', '=', False)]}"/>
+                                <div attrs="{'invisible': [('propagate_date', '=', False)]}">
+                                    <field name="propagate_date_minimum_delta" class="oe_inline"/> day(s)
+                                </div>
+                            </group>
```

Comment:
add a basic test: create two chained moves, enable propate + minimum delta, change 

with two chained moves
1) enable propagate, change expected dat
2) enabe propagate, action_done on the first
3) disable propagate, change expected dat
4) disable propagate, action_done on the first